From d7fd64c03e0027b069f9cd9557d6d42ade758cd7 Mon Sep 17 00:00:00 2001
From: Alexandru Ardelean <alexandru.ardelean@analog.com>
Date: Mon, 14 Oct 2019 11:33:03 +0300
Subject: [PATCH 15/16] [media] adv7604: fix deep color mode

This seems to have been broken in some update.
This patch brings back support for Deep Color mode.

Fixes: b1176dc46cad4 ("[media] adv7604: sync driver to Xilinx's version")
Signed-off-by: Alexandru Ardelean <alexandru.ardelean@analog.com>
(cherry picked from commit 690b286400743b58ba2797fe5280e779bfef0500)
---
 drivers/media/i2c/adv7604.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/adv7604.c b/drivers/media/i2c/adv7604.c
index ba9dd2b6825e..166eba347396 100644
--- a/drivers/media/i2c/adv7604.c
+++ b/drivers/media/i2c/adv7604.c
@@ -1625,7 +1625,7 @@ static int adv76xx_query_dv_timings(struct v4l2_subdev *sd,
 
 		bt->width = w;
 		bt->height = h;
-		bt->pixelclock = info->read_hdmi_pixelclock(sd);
+		bt->pixelclock = adv76xx_read_hdmi_pixelclock(sd);
 		bt->hfrontporch = hdmi_read16(sd, 0x20, info->hfrontporch_mask);
 		bt->hsync = hdmi_read16(sd, 0x22, info->hsync_mask);
 		bt->hbackporch = hdmi_read16(sd, 0x24, info->hbackporch_mask);
-- 
2.17.1

